cfspopcon/algorithms/__init__.py

```diff
@@ -53,22 +55,6 @@ def get_algorithm(algorithm: Union[Algorithms, str]) -> Union[Algorithm, Composi
 
 
 __all__ = [
-    "calc_beta",
-    "calc_core_radiated_power",
-    "calc_extrinsic_core_radiator",
-    "calc_fusion_gain",
-    "calc_geometry",
-    "calc_heat_exhaust",
-    "calc_ohmic_power",
-    "calc_peaked_profiles",
-    "calc_plasma_current_from_q_star",
-    "calc_power_balance_from_tau_e",
-    "predictive_popcon",
-    "calc_q_star_from_plasma_current",
-    "two_point_model_fixed_fpow",
-    "two_point_model_fixed_qpart",
-    "two_point_model_fixed_tet",
-    "calc_zeff_and_dilution_from_impurities",
```

Comment:
This means non of these show up by default in the docs. 
Why remove them here?

Reply:
How do I do this?